Core/src/Geometry/DetectorVolume.cpp

```diff
@@ -13,11 +13,16 @@
 #include "Acts/Geometry/VolumeBounds.hpp"
 #include "Acts/Geometry/detail/DetectorVolumeUpdators.hpp"
 #include "Acts/Surfaces/Surface.hpp"
+#include "Acts/Utilities/BoundingBox.hpp"
 #include "Acts/Utilities/Enumerate.hpp"
 #include "Acts/Utilities/Helpers.hpp"
 
 #include <cassert>
 
+using BoundingBox =
+    Acts::AxisAlignedBoundingBox<Acts::Experimental::DetectorVolume,
+                                 Acts::ActsScalar, 3>;
```

Comment:
I don't think you need to repeat the typedef here.

Reply:
Done.